### DISCUSSION TITLE
Filter notifications by issue / pull request status (open, closed)

### DISCUSSION THREAD

**Discussion by danirod** [community/community#4796] [Projects and Issues]
Even though it took me a while to start using the Notifications dashboard (github.com/notifications), I have fallen in love with this tool, it helps me a lot to manage what's pending to be done, both on my personal projects and for work, such as keeping track of the opened bugs, see updates on repositories I'm watching, or manage review requests from my teammates, for instance.

However, sometimes I spend too much time cleaning up notifications that are not actionable anymore. For instance, when a pull request opened by Dependabot is superseded by another, the old pull request will get closed, but the notification will stay in my inbox until I mark it as done. Because I have Dependabot enabled for many repositories, these old notifications are mixed in between opened ones and it is difficult to "mark as done" the older ones.

My question is, it seems that there is no way to filter notifications by whether the pull request is opened, merged or closed, right? I have read the help page for filters and it doesn't seem possible at the moment to have something like `author:app/dependabot is:issue-or-pull-request [¿status:closed?]` to make it easier to "select all + mark as done" the notifications that are not actionable anymore. I would like to share this as an idea in case it's not available at the moment.

**Comment by iamthen0ise** [comment]
+1

Crawling thru closed/solved issues in notification is not a lovely routine.

**Comment by andyfeller** [comment]
I'm on the fence about whether a new field is added _(`pull-request-status` / `issue-status`)_ or if `is:` is enhanced since it serves multiple purposes already.

I would like to include:
1. knowing when a pull request is merged
2. having the _"Unread"_ button from the Inbox on all subsequent pages

**Comment by CanRau** [comment]
Wow thanks so much was trying to figure out how to filter Dependabot and the obvious and auto completed `author:dependabot[bot]` wouldn't work 😬 How did you figure the `author:app/dependabot` out?

**Comment by kevin--** [comment]
big upvote for this one. I need to clear out dozens of merged PRs every week. Super annoying

**Comment by billyvg** [comment]
Dupe here as well: https://github.com/orgs/community/discussions/15591
